smdebug/tensorflow/keras.py

```diff
@@ -668,7 +705,7 @@ def _on_any_batch_begin(self, batch, mode, logs=None):
             if (is_tf_version_2x() and tf.executing_eagerly()) or self._validate_exec_function(
                 self._get_exec_function(mode)
             ):
-                self._wrap_model_with_input_output_saver()
+                # self._wrap_model_with_input_output_saver()
```

Comment:
nit: If this line is not needed, can it be removed?

Reply:
Will do. Thanks!